repos/roslyn.proj

```diff
@@ -27,15 +30,26 @@
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.Compilers.nuspec" />
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.VisualBasic.nuspec" />
     <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.Net.Compilers.netcore.nuspec" />
-    <NuSpecFiles Include="$(ProjectDirectory)/src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec" />
   </ItemGroup>
 
+  <Target Name="UpdateCLIVersion"
```

Comment:
Could this be in `dir.targets` with the json and props paths parameterized? (But we can put the move off until we use it to remove patches in other repos too.)

Reply:
I'd rather leave this here for now and move it when we need it for others.